pyproject.toml

```
@@ -107,7 +107,7 @@ module = [
     "rotkehlchen.tests.api.test_history",
     "rotkehlchen.tests.api.test_icons",
     "rotkehlchen.tests.api.test_ignored_actions",
-    "rotkehlchen.tests.api.test_liquidy",
+    "rotkehlchen.tests.api.test_liquidity",
```

This is a file name. 

```suggestion
    "rotkehlchen.tests.api.test_liquity",
```